library/node

```diff
@@ -113,10 +98,10 @@ Architectures: amd64, ppc64le, s390x, arm64v8, arm32v7, i386
 GitCommit: 2ecc9e8579f519ae3d267b5b497b8c04d6c7040d
 Directory: 10/stretch
 
-Tags: 10.1.0-wheezy, 10.1-wheezy, 10-wheezy, wheezy
-Architectures: amd64
-GitCommit: 2ecc9e8579f519ae3d267b5b497b8c04d6c7040d
-Directory: 10/wheezy
+Tags: ., -images
+Architectures: amd64, ppc64le, s390x, arm64v8, arm32v7, i386
+GitCommit: 
+Directory: official-images
```

Comment:
@LaurentGoderre This part is odd. I think what's happening is the `generate-stackbrew-library.sh` is somehow including the locally checked out `official-images` repo and because that repo has a `Docekrfile`, it's including it in the manifest generation.

Reply:
OOOOH! you're right!!!....wonder if I should try ignoring that folder or in Travis, clone in the parent directory. What do you think?

Reply:
Hmm. I think maybe it's better to clone the directory somewhere else if possible.

Reply:
fwiw, I don't think we would have caught this without having the actual PR happen. Nice to see that part working tho!